File: Android/Vine-5.50.1/Decompilation/app/src/main/java/co/vine/android/scribe/model/MobileRadioDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
/* loaded from: classes.dex */
public class MobileRadioDetails {

    @JsonField(name = {"mobile_network_operator_code"})
    public String mobileNetworkOperatorCode;

    @JsonField(name = {"mobile_network_operator_country_code"})
    public String mobileNetworkOperatorCountryCode;

    @JsonField(name = {"mobile_network_operator_iso_country_code"})
    public String mobileNetworkOperatorIsoCountryCode;

    @JsonField(name = {"mobile_network_operator_name"})
    public String mobileNetworkOperatorName;

    @JsonField(name = {"mobile_sim_provider_code"})
    public String mobileSimProviderCode;

    @JsonField(name = {"mobile_sim_provider_iso_country_code"})
    public String mobileSimProviderIsoCountryCode;

    @JsonField(name = {"mobile_sim_provider_name"})
    public String mobileSimProviderName;

    @JsonField(name = {"radio_status"})
    public String radioStatus;

    @JsonField(name = {"signal_strength"})
    public Integer signalStrength;
}

File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/MobileRadioDetails.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class MobileRadioDetails
{
	[JsonProperty("mobile_network_operator_code")]
	public string MobileNetworkOperatorCode { get; set; }

	[JsonProperty("mobile_network_operator_iso_country_code")]
	public string MobileNetworkOperatorIsoCountryCode { get; set; }

	[JsonProperty("mobile_network_operator_name")]
	public string MobileNetworkOperatorName { get; set; }
}
